devicetypes/smartthings/smartpower-outlet.src/smartpower-outlet.groovy

```diff
@@ -128,8 +136,9 @@ def refresh() {
 }
 
 def configure() {
-	// Device-Watch allows 2 check-in misses from device
-	sendEvent(name: "checkInterval", value: 60 * 12, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
+	// Device-Watch allows 3 check-in misses from device (plus 3 min lag time)
+	sendEvent(name: "checkInterval", value: 3 * 60 * 60 + 1 * 60, displayed: false, data: [protocol: "zigbee"])
```

Comment:
I understand. 

The checkInterval calculation doesn't look right. The report config is 5 minutes, so shouldn't it be `3 * 5 * 60 + 3 * 60` in order to match the comment?